packages/cisco_duo/kibana/dashboard/cisco_duo-4eb684b0-0fbe-11ec-8b4b-67126a72b1d4.json

```
@@ -0,0 +1,27 @@
+{
+  "attributes": {
+    "description": "",
+    "hits": 0,
+    "kibanaSavedObjectMeta": {
+      "searchSourceJSON": "{\"filter\":[],\"query\":{\"language\":\"kuery\",\"query\":\"\"}}"
+    },
+    "optionsJSON": "{\"hidePanelTitles\":false,\"syncColors\":false,\"useMargins\":true}",
+    "panelsJSON": "[{\"embeddableConfig\":{\"enhancements\":{}},\"gridData\":{\"h\":23,\"i\":\"56de1cf8-48cc-429a-a54f-ca4f4e653590\",\"w\":21,\"x\":0,\"y\":0},\"panelIndex\":\"56de1cf8-48cc-429a-a54f-ca4f4e653590\",\"panelRefName\":\"panel_56de1cf8-48cc-429a-a54f-ca4f4e653590\",\"type\":\"visualization\",\"version\":\"7.15.0-SNAPSHOT\"}]",
+    "timeRestore": false,
+    "title": "[ Cisco Duo ] Administrator Logs",
```

```suggestion
    "title": "[Cisco Duo] Administrator Logs",
```

Please do a find/replace to trim those spaces in all of the titles.